src/Schemes/CoveredSchemes/Objects/Attributes.jl

```diff
@@ -18,16 +18,21 @@ base_ring(X::AbsCoveredScheme) = base_ring(underlying_scheme(X))
 @Markdown.doc """
     coverings(X::AbsCoveredScheme)
 
-Returns the available coverings for ``X``.
+On a `CoveredScheme` ``X`` this returns a list of the internally stored 
+`Covering`s of ``X``.
```

Comment:
Please follow
https://docs.julialang.org/en/v1/manual/documentation/
when writing documentation. 
In this particular example it is:
```suggestion
Return the list of internally stored `Covering`s of ``X``.
```
There is no need to say that `X` is a covered scheme, since that is what the signature says.

Reply:
This applies to most of the other docstrings as well.